### ISSUE TITLE
Fake street teams break match lineup (again)

### ISSUE THREAD

**Issue by minj** [minj/foxtrick#1442]
**From:** murko
**PostID:** [16896702.408](https://www.hattrick.org/goto.ashx?path=%2FForum%2FRead.aspx%3Ft%3D16896702%26n%3D408%26v%3D0)
**To:** LA-MJ
**Re:** [16896702.407](https://www.hattrick.org/goto.ashx?path=%2FForum%2FRead.aspx%3Ft%3D16896702%26n%3D407%26v%3D0)
**Datetime:** 2016-09-28 21:56
**Message:**

Sundays friendly **(Good)** :

[(http://easycaptures.com/fs/uploaded/1072/4800...0.png)](http://easycaptures.com/fs/uploaded/1072/4800793880.png)

[ ![Įkeltas paveikslėlis](http://easycaptures.com/fs/uploaded/1072/4800793880.png) ](http://easycaptures.com/fs/uploaded/1072/4800793880.png)

Todays cup game **(Bad)** :

[(http://easycaptures.com/fs/uploaded/1072/0839...9.png)](http://easycaptures.com/fs/uploaded/1072/0839893949.png)

[ ![Įkeltas paveikslėlis](http://easycaptures.com/fs/uploaded/1072/0839893949.png) ](http://easycaptures.com/fs/uploaded/1072/0839893949.png)

Is everything breaking because the opponent is one of those "fake teams" (cup fillers) and there are no real data on them?

---

> Foxtrick 0.17.9.287 beta-41417a2 mozilla - Stage: false - Skin: normal - en-GB - ltr - 1920 x 1050 - Mozilla/5.0 (Windows NT 10.0; rv:49.0) Gecko/20100101 Firefox/49.0 - Win32

**Comment by minj** [comment]
**From:** LA-MJ
**PostID:** [16934800.621](https://www.hattrick.org/goto.ashx?path=%2FForum%2FRead.aspx%3Ft%3D16934800%26n%3D621%26v%3D0)
**To:** HT-Bodin
**Re:** [16934800.1](https://www.hattrick.org/goto.ashx?path=%2FForum%2FRead.aspx%3Ft%3D16934800%26n%3D1%26v%3D0)
**Datetime:** 2016-10-03 22:07
**Message:**

I have a curious case for you that might indicate something weird going on in the back-end.

Guess what happens on the first week every season and makes me mad every single time...

> People play those wretched fake street teams

---

See this match:

[(/Club/Matches/Match.aspx?matchID=583957115&So...#tab2)](https://www.hattrick.org/goto.ashx?path=%2FClub%2FMatches%2FMatch.aspx%3FmatchID%3D583957115%26SourceSystem%3DHattrick%26TeamId%3D11598%26UpdateViewedReport%3DFalse%23tab2)

A **human** player is injured and leaves @18'

However, the underling JSON object says `player.ToMin=17`

Similarly:

[(/Club/Matches/Match.aspx?matchID=583956843&So...#tab2)](https://www.hattrick.org/goto.ashx?path=%2FClub%2FMatches%2FMatch.aspx%3FmatchID%3D583956843%26SourceSystem%3DHattrick%26TeamId%3D-1%26UpdateViewedReport%3DFalse%23tab2)

A **human** player is injured and leaves @8'

However, the underling JSON object says `player.ToMin=7`

Compare with: [(/Club/Matches/Match.aspx?matchID=583956840&So...#tab2)](https://www.hattrick.org/goto.ashx?path=%2FClub%2FMatches%2FMatch.aspx%3FmatchID%3D583956840%26SourceSystem%3DHattrick%26TeamId%3D-1%26UpdateViewedReport%3DFalse%23tab2)

A **fake** player is injured and leaves @42'. Underlying JSON matches.

Btw, timed subs and red cards appear to be fine:

[(583956841)](https://www.hattrick.org/goto.ashx?path=%2FClub%2FMatches%2FMatch.aspx%3FmatchID%3D583956841) (p. s. what a curious match!)

[(583956846)](https://www.hattrick.org/goto.ashx?path=%2FClub%2FMatches%2FMatch.aspx%3FmatchID%3D583956846)

[(583956842)](https://www.hattrick.org/goto.ashx?path=%2FClub%2FMatches%2FMatch.aspx%3FmatchID%3D583956842)

[(331203238)](https://www.hattrick.org/goto.ashx?path=%2FClub%2FMatches%2FMatch.aspx%3FmatchID%3D331203238)

To sum up: only matches against street teams are affected. Only human player injuries are affected.

---

**TL;DR:**

See here: [(374692205)](https://www.hattrick.org/goto.ashx?path=%2FClub%2FPlayers%2FPlayer.aspx%3FplayerId%3D374692205). It says he played 7'. See [(583956843)](https://www.hattrick.org/goto.ashx?path=%2FClub%2FMatches%2FMatch.aspx%3FmatchID%3D583956843). It says he played 8'.

I have a feeling you have robbed these teams of 1' worth of training :D

> and broke Foxtrick in the bargain:P

P. S. you know where to send the Pulitzer ;D

**Comment by minj** [comment]
**From:** HT-Gusy
**PostID:** [16934800.623](https://www.hattrick.org/goto.ashx?path=%2FForum%2FRead.aspx%3Ft%3D16934800%26n%3D623%26v%3D0)
**To:** LA-MJ
**Re:** [16934800.621](https://www.hattrick.org/goto.ashx?path=%2FForum%2FRead.aspx%3Ft%3D16934800%26n%3D621%26v%3D0)
**Datetime:** 2016-10-05 13:07
**Message:**

Wow, this was a very nasty bug. I've fixed it and will release it in time for the weekend matches.

From what I saw, it could happen at any match, so I don't know (and don't think it's worth investigating) why it happened only in those particular cases.

And, for the record, what's wrong is the event minute and not the player data. So technically they didn't lose any training ;) Btw, also the player who comes in has (or should have) the "wrong" FromMin.
